server/client_manager.py

```diff
@@ -1046,6 +1049,9 @@ def change_area(self, area, password=""):
                         c.send_ooc(ex)
                         c.unfollow()
                         return
+            # Last error check got done above.
+
+            self.clear_area_char_links(old_area)
```

Comment:
Oh, this also defines new char_links? Maybe "refresh" instead of "clear" would be better?

Reply:
Yup. It would be more ideal.